### ISSUE TITLE
Unable to start CoreWiki due to font awesome source error in libman.js on macOS

### ISSUE THREAD

**Issue by tscholze** [csharpfritz/CoreWiki#328]
Could not start CoreWiki from Terminal or VS on macOS.
It would be great if somebody could help me to get the Core Wiki started.

Thanks you very much!

**Information**
OS: macOS 10.14
dotnet: 2.1.403
CoreWiki: `d1bacf7cf0727c9231f824a2caf0a11a9c569443`

**Error**
```
libman.json : error LIB002: The "node_modules/@fortawesome/fontawesome-free/" library could not be resolved by the "filesystem" provider
```

**Related source** (libman.json)
```
{
    "provider": "filesystem",
    "library": "node_modules/@fortawesome/fontawesome-free/",
    "destination": "wwwroot/lib/fontawesome/",
    "files": [
        "css/all.css",
        "css/all.min.css",
        "js/all.js",
        "js/all.min.js"
    ]
}
```

**Comment by danielmpetrov** [comment]
I might be stating the obvious, but is fortawesome actually present at that path?

If not, try running ```npm install``` in the web project.

**Comment by tscholze** [comment]
@danielmpetrov.
Yes I had, but from the wrong directory. I executed the command from the  root dir `CoreWiki`. But I had to call it form `CoreWiki/CoreWiki`.

Thanks for helping me out!

**Idea**
Maybe we could add a section to the Readme file on how to start the project (on different operation systems). 

The steps could also contain an information how to proceed if a new user sees this view. :)
![bildschirmfoto 2018-10-23 um 10 12 03](https://user-images.githubusercontent.com/5583469/47345909-38c5b480-d6ac-11e8-9f8f-bc6510082d5f.png)
